compiler/src/dmd/expression.h

```diff
@@ -52,6 +53,8 @@ namespace dmd
     Expression *expressionSemantic(Expression *e, Scope *sc);
     // in typesem.d
     Expression *defaultInit(Type *mt, const Loc &loc, const bool isCfile = false);
+    // in enumsem.d
+    Expression *getDefaultValue(EnumDeclaration *ed, const Loc &loc);
```

Comment:
[enum.h](https://github.com/dlang/dmd/blob/stable/compiler/src/dmd/enum.h#L20) seems like a more appropriate location for this declaration.

Reply:
Though I do see why it would be put here as well.